src/sentry/api/endpoints/organization_events_trace.py

```diff
@@ -371,10 +399,10 @@ def full_dict(
         event_id = self.event["id"]
         # We're in a loop!
         if event_id in visited:
-            return
+            return None
         else:
             visited.add(self.event["id"])
-        result = self.to_dict()
+        result = cast(FullResponse, self.to_dict())
```

Comment:
Is this casting a `LightResponse` to a `FullResponse`? Is this right?

Reply:
Yeah, I a FullResponse shares a lot with a LightResponse so this is just extending it. Probably could rewrite this to be a bit cleaner, but we aren't using light responses anymore in environments where spans are enabled